src/Analysis/GWASV2/Steps/SelectOutcome/SelectOutcome.jsx

```diff
@@ -59,13 +70,27 @@ const SelectOutcome = ({ dispatch, studyPopulationCohort, outcome }) => {
   };
 
   // Outputs the JSX for the component:
-  return <div>{determineSelectOutcomeJsx()}</div>;
+  return (
+    <div className='GWASUI-row'>
+      <div className='GWASUI-double-column'>{determineSelectOutcomeJsx()}</div>
+      <div className='GWASUI-column GWASUI-card-column'>
+        <CovariatesCardsList
+          covariates={covariates}
+          deleteCovariate={(chosenCovariate) => dispatch({
+            type: ACTIONS.DELETE_COVARIATE,
+            payload: chosenCovariate,
+          })}
+        />
+      </div>
+    </div>
+  );
 };
 
 SelectOutcome.propTypes = {
   dispatch: PropTypes.func.isRequired,
   studyPopulationCohort: PropTypes.object.isRequired,
   outcome: PropTypes.object,
+  covariates: PropTypes.object.isRequired,
```

Comment:
I would prefer this to be optional, with a default value of `[]`. IMO the required items should be just the ones that must have a non-empty value.

Reply:
Sure, updated now to use a Prop Type to define a default value of []